addOns/webuipoc/src/main/pocs/reactWebUI/src/Components/Accordion/AccordionData.jsx

```diff
@@ -11,7 +11,13 @@ const Accordion = ({ site, fetchChildren }) => {
         }
         setAccordionOpen(!isAccordionOpen);
     }
-  
+    const getDisplayName = (name) => {
```

Comment:
I'm good with this (and the method) to be merged as is but it's not how it should be implemented in the end, this logic should be in ZAP only.